src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -624,25 +868,24 @@ static void fillRefKindsBuilder(ImmutableArray<RefKind> argumentRefKindsOpt, Imm
 
         // temporariesBuilder will be null when factory is null.
         private static bool CanSkipRewriting(
```

Comment:
>CanSkipRewriting [](http://example.com/codeflow?start=28&length=16)

I think we shouldn't change this function and it should only deal with reordering of arguments. The handler rewrite should be a separate pass through the arguments with its own stores, etc.